changelog/2505.trivial

```diff
@@ -0,0 +1 @@
+Return the oldval in monkeypatch for easier wrapping.
```

Comment:
This isn't very understandable I think - what about "Return the previous value in monkeypatch.setattr"?